App.tsx

```diff
@@ -20,10 +20,17 @@ const App = () => {
         <Pressable
           disabled={!didConfig}
           style={[styles.button]}
-          onPress={() => startFitnessAssessment()}>
+          onPress={() => startFitnessAssessment(AssessmentTypes.Fitness, true, null)}>
           <Text style={styles.textStyle}>Start Assessment</Text>
         </Pressable>
 
+        <Pressable
+          disabled={!didConfig}
+          style={[styles.button]}
+          onPress={() => startFitnessAssessment(AssessmentTypes.Custom, true, "YOUR_CUSTOM_ASSESSMENT")}>
```

Comment:
@SencyNetanel the interface name here doesn't make sense, startFitnessAssessment
please change it to a global one - startAssessment